docs/swagger_v3/stats.spec.yaml

```diff
@@ -394,6 +394,61 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
+  /stats/blocks_difficulty:
```

Comment:
Can we name this `/stats/difficulty`? There's no other difficulty related concept other than blocks